packages/ledgerstate/branch.go

```diff
@@ -264,6 +254,19 @@ func (b BranchIDs) Slice() (list []BranchID) {
 	return
 }
 
+// IsMasterBranch returns true if the BranchIDs are either empty or contain only the MasterBranch.
+func (b BranchIDs) IsMasterBranch() (isMasterBranch bool) {
+	if len(b) == 0 {
+		return true
+	}
```

Comment:
Isn't that confusing that it tells you it is master branch but it is actually empty?